test/cogctl/actions/bundles.exs

```diff
@@ -62,7 +80,8 @@ defmodule Cogctl.Actions.Bundles.Test do
 
   test "bundles info returns an error when args are missing" do
     assert run("cogctl bundles info") =~ ~r"""
-    ERROR: "Missing required arguments"
+    Usage: [\s\S]*\
+    cogctl: Missing required arguments: 'bundle'
```

Comment:
It's impossible for the user to tell which stream a message was written to without redirecting stdout or stderr or configuring some sort of color coding (stdout in green, stderr in red, etc). Given this is the case I think it's still useful to flag errors as errors to make the user's life easier.